### ISSUE TITLE
Disable masquerading for production

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/myaccount#116]

**Comment by cdmo** [comment]
as a result of our my account checkin today:

* @cdmo will make a flow chart of proposed
* @banukutlu will review

**Comment by cdmo** [comment]
Flowcharts for authentication:

* [As it is now](https://psu.box.com/s/scb0h8nze1rbs6a98rcp2t1x9wvj6b4s)
* [Potential solution](https://psu.box.com/s/4x7kb9o73ddop9li2vyfwqez7dk0os38)

**Comment by banukutlu** [comment]
> Flowcharts for authentication:
> 
> * [As it is now](https://psu.box.com/s/scb0h8nze1rbs6a98rcp2t1x9wvj6b4s)
> * [Potential solution](https://psu.box.com/s/4x7kb9o73ddop9li2vyfwqez7dk0os38)

Thanks @cdmo for writing these up, your proposed solution makes sense to me. One question I have is the order of auths, wondering if it would matter if we do the myaccount webaccess authentication prior to Sirsi one?

**Comment by cdmo** [comment]
I don't think it would 🤔 Do you have a concern?

**Comment by banukutlu** [comment]
> I don't think it would 🤔 Do you have a concern?

No, I do not, was just trying to figure if the order matters.

**Comment by cdmo** [comment]
Hey @banukutlu I just now noticed that you removed yourself from this one. Are you done looking at it then? Should we go forward with the proposed solution you think? This means closing this issue and making a new issue for adding WebAccess to the application itself (secondary layer of webaccess). It has been some time since I looked at this but I recall vaguely you wanted to look at it closer before we moved forward with the plan.

**Comment by banukutlu** [comment]
> Hey @banukutlu I just now noticed that you removed yourself from this one. Are you done looking at it then? Should we go forward with the proposed solution you think? This means closing this issue and making a new issue for adding WebAccess to the application itself (secondary layer of webaccess). It has been some time since I looked at this but I recall vaguely you wanted to look at it closer before we moved forward with the plan.

Yes, I think your proposed solution makes sense and we should implement it.

**Comment by cdmo** [comment]
ok cool, see #200
